src/extensions/common/CanvasWidget/CanvasPainter.ts

```diff
@@ -129,11 +129,16 @@ export class CanvasPainter {
         }
     }
     useOffscreenCanvas(W: number, H: number) {
-        const c = new OffscreenCanvas(W, H)
-        this._offscreenCanvas = c
-        const cc = c.getContext('2d')
-        if (!cc) throw Error('Unexpected')
-        this._context2D = cc
+        try {
```

Comment:
Be more careful about the OffscreenCanvas creation so that we don't crash the app